wherehows-web/app/typings/rsvp.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'rsvp' {
```

Comment:
Why do you need to do this? the types in  `@types/rsvp` should be sufficient or maybe need to update the version installed?

Reply:
Yes, internally the library needs this...

Reply:
Posted a question in Discord. Will wait to see if there is an easier way.

Reply:
I am reverting this change and manually add the definitions manually.